src/main/java/com/deavensoft/paketomat/dispatcher/DispatcherServiceImpl.java

```diff
@@ -47,10 +50,11 @@ public class DispatcherServiceImpl implements DispatcherService {
 
     @Autowired
     public DispatcherServiceImpl(@Qualifier("dispatcher") DispatcherRepository dispatcherRepository,
-                                 UserService userService, EmailService emailService) {
+                                 UserService userService, EmailService emailService, CenterService centerService) {
```

Comment:
@RequiredArgsConstructor generates a constructor requiring an argument for the final and @NonNull fields.

- [ ] Fix it.